setup.cfg

```diff
@@ -38,32 +38,17 @@ select =
 [mypy]
 ignore_missing_imports = True
 
-[mypy-cudf._lib.*]
```

Comment:
I'm now `--exclude`ing these modules, rather than parsing and then ignoring errors.

Reply:
Smart. Should we do the same for the _version files?

Reply:
Good idea, done.